includes/data-stores/class-wc-product-data-store-cpt.php

```diff
@@ -680,7 +680,7 @@ protected function update_version_and_type( &$product ) {
 	 */
 	protected function clear_caches( &$product ) {
 		wc_delete_product_transients( $product->get_id() );
-		wp_cache_delete( 'product-' . $product->get_id(), 'products' );
+		wp_cache_delete( 'object-' . $product->get_id(), 'products' );
```

Comment:
👍 Just to point out the name doesn't matter for clearing since we clear by group. I just changed it to `object` so it would apply to all object types in the `WC_Data` class.